README.md

```diff
@@ -4,5 +4,101 @@
 [![CI](https://github.com/AIRCentre/SARProcessing.jl/actions/workflows/CI.yml/badge.svg)](https://github.com/AIRCentre/SARProcessing.jl/actions/workflows/CI.yml)
 [![Codecov](https://codecov.io/gh/AIRCentre/SARProcessing.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/AIRCentre/SARProcessing.jl)
 
+<p align="center">
+<img src="figures/detect_ship.png" height="200"><br>
```

Comment:
The image is now place at "examples/figures/detect_ship.png"

Reply:
Should I create another folder called 'figures' and place it there?

Reply:
I would just update the path to "examples/figures/detect_ship.png".
The important thing is that the image location matches the path so the image is shown when viewing the readme on github
You can see that the image is missing when opening the branch https://github.com/AIRCentre/SARProcessing.jl/tree/igaszczesniak-patch-1

Reply:
Oki should be good rn

Reply:
Perfect. I have approved the PR now. You can merge the branch when the tests are done. Please chose "Squash and merge". Then the PR will just look like a single commit after merge :)